src/fsharp/fsi/fsi.fs

```diff
@@ -1641,18 +1663,124 @@ module internal MagicAssemblyResolution =
                stopProcessingRecovery e range0
                null
 
+#if NETSTANDARD
+        let probingFileNames (name: string) =
```

Comment:
Include link to this algorithm.

Reply:
https://github.com/dotnet/coreclr/blob/9773db1e7b1acb3ec75c9cc0e36bd62dcbacd6d5/src/System.Private.CoreLib/shared/System/Runtime/Loader/LibraryNameVariation.Unix.cs

And added comment.